src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -468,7 +473,7 @@ internal static void PopulatePruningEnabledTelemetry(PackageSpec project, Teleme
                     if (cacheFileAge.HasValue) { telemetry.TelemetryEvent[NoOpCacheFileAgeDays] = cacheFileAge.Value.TotalDays; }
 
                     return (new NoOpRestoreResult(
-                            _success,
+                            success,
```

Comment:
This was probably an oversight when I introduced no-op, but this is *always* true by the time we get to this point. 
No need to change it.